src/data/repositories/AnalyticsD2Repository.ts

```diff
@@ -0,0 +1,272 @@
+// @ts-nocheck
+import { Maybe } from "./../../utils/ts-utils";
+import { AnalyticsResponse, D2Api } from "../../types/d2-api";
+import { AnalyticsRepository } from "../../domain/repositories/AnalyticsRepository";
+import { apiToFuture, FutureData } from "../api-futures";
+import { RTSL_ZEBRA_PROGRAM_ID } from "./consts/DiseaseOutbreakConstants";
+import _ from "../../domain/entities/generic/Collection";
+import { Future } from "../../domain/entities/generic/Future";
+import {
+    IndicatorsId,
+    NB_OF_ACTIVE_VERIFIED,
+    NB_OF_CASES,
+    NB_OF_DEATHS,
+} from "./consts/AnalyticsConstants";
+import moment from "moment";
+import { DiseaseOutbreakEventBaseAttrs } from "../../domain/entities/disease-outbreak-event/DiseaseOutbreakEvent";
+
+export type ProgramIndicatorBaseAttrs = {
+    id: string;
+    event: string;
+    province: string;
+    duration: string;
+    manager: string;
+    cases: string;
+    deaths: string;
+    era1: string;
+    era2: string;
+    era3: string;
+    era4: string;
+    era5: string;
+    era6: string;
+    era7: string;
+    detect7d: string;
+    notify1d: string;
+    respond7d: string;
+    creationDate: string;
+    suspectedDisease: string;
+    hazardType: string;
+};
+
+interface DiseaseEntry {
+    disease: string;
+    total: number;
+}
+
+interface HazardEntry {
+    hazard: string;
+    total: number;
+}
+
+export type DiseaseTotalAttrs = DiseaseEntry | HazardEntry;
+
+export class AnalyticsD2Repository implements AnalyticsRepository {
```

Comment:
analytics, indicators etc are DHIS concepts. As per clean architecture the repository should only deal with domain entities. So you can you rename the class, file and functions to not have any references to DHIS concepts but instead refer to Zebra concepts - performance overview table, performance overview repository etc.